test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -130,38 +130,38 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
       ~s(sidebarNodes={"extras":[{"id":"","title":"","group":"","headers":[{"id":"Foo","anchor":"foo"}]}]})
   end
 
-  test "synopsis" do
-    assert Templates.synopsis(nil) == nil
-    assert Templates.synopsis("") == ""
-    assert Templates.synopsis(".") == ""
-    assert Templates.synopsis(".::.") == ""
-    assert Templates.synopsis(" .= .: :.") == ".="
-    assert Templates.synopsis(" Description: ") == "Description"
-    assert Templates.synopsis("abcd") == "abcd"
-    assert_raise FunctionClauseError, fn ->
-      Templates.synopsis(:abcd)
-    end
-  end
+  # test "synopsis" do
+  #   assert Templates.synopsis(nil) == nil
+  #   assert Templates.synopsis("") == ""
+  #   assert Templates.synopsis(".") == ""
+  #   assert Templates.synopsis(".::.") == ""
+  #   assert Templates.synopsis(" .= .: :.") == ".="
+  #   assert Templates.synopsis(" Description: ") == "Description"
+  #   assert Templates.synopsis("abcd") == "abcd"
+  #   assert_raise FunctionClauseError, fn ->
+  #     Templates.synopsis(:abcd)
+  #   end
+  # end
 
-  test "synopsis should not end have trailing periods or semicolons" do
-    doc1 = """
-    Summaries should not be displayed with trailing punctuation . :
+  # test "synopsis should not end have trailing periods or semicolons" do
+  #   doc1 = """
+  #   Summaries should not be displayed with trailing punctuation . :
 
-    ## Example
-    """
+  #   ## Example
+  #   """
 
-    doc2 = """
-    Example function: Summary should not display trailing puntuation :.
+  #   doc2 = """
+  #   Example function: Summary should not display trailing puntuation :.
 
-    ## Example:
-    """
+  #   ## Example:
+  #   """
 
-    assert Templates.synopsis(doc1) ==
-      "Summaries should not be displayed with trailing punctuation"
+  #   assert Templates.synopsis(doc1) ==
+  #     "Summaries should not be displayed with trailing punctuation"
 
-    assert Templates.synopsis(doc2) ==
-      "Example function: Summary should not display trailing puntuation"
-  end
+  #   assert Templates.synopsis(doc2) ==
+  #     "Example function: Summary should not display trailing puntuation"
+  # end
```

Comment:
Yes, let's revert the synopsis change for now. We can always add the feature if necessary later.